Samples/WindowsML/WindowsML-Samples.sln

```diff
@@ -74,8 +74,10 @@ Global
 		{A1B2C3D4-5678-9012-3456-789012345678}.Release|x64.Build.0 = Release|x64
 		{C9BD7502-33EE-4826-A6D5-2F6B4AC90346}.Debug|ARM64.ActiveCfg = Debug|ARM64
 		{C9BD7502-33EE-4826-A6D5-2F6B4AC90346}.Debug|ARM64.Build.0 = Debug|ARM64
+		{C9BD7502-33EE-4826-A6D5-2F6B4AC90346}.Debug|ARM64.Deploy.0 = Debug|ARM64
```

Comment:
On the device where I was working on this change, I have been unable to deploy the packaged WinUI sample. I'm hitting what appears to be the bug here: https://github.com/microsoft/microsoft-ui-xaml/issues/6378

Haven't been able to find a workaround and will need to test this on another device.